lib/matplotlib/mpl-data/stylelib/dark_background.mplstyle

```diff
@@ -21,3 +21,10 @@ figure.edgecolor: black
 savefig.facecolor: black
 savefig.edgecolor: black
 
+### Boxplots
+boxplot.boxprops.color: white
+boxplot.capprops.color: white
+boxplot.flierprops.color: white
+boxplot.flierprops.markeredgecolor: white
+boxplot.meanprops.markeredgecolor: white
```

Comment:
```suggestion
```

Per @timhoffm 's suggestion.  I think it is better to track the color cycle for these so they stay highlighted from the other lines.